packages/node_modules/@node-red/nodes/core/function/90-exec.html

```diff
@@ -53,6 +55,8 @@
         defaults: {
             command: {value:""},
             addpay: {value:false},
+            addpayValue: {value:"payload"},
+            addpayValueType: {value:"msg"},
```

Comment:
if `msg` is the only valid value, then there is no need to add a property.

if we decide to add other types in the future we can then add a property to store the type.

Reply:
Thank you for the information. I removed the `addpayValueType` in the commit, https://github.com/node-red/node-red/pull/2908/commits/cb72d5100eeff889985be6f28f5fe9638d4299b5.